### ISSUE TITLE
problem with crash

### ISSUE THREAD

**Issue by RealHumanWithHair** [dreamstalker/rehlds#589]

**Comment by In-line** [comment]
Indeed very useful.. For team of professional espers. From what version to what did you update? When does crash happen? Show meta list, amx_plugins, version, game version. What part of your server did you update? (only swds.dll? or hlds.exe too?).

**Comment by RealHumanWithHair** [comment]
Current:
ReHLDS version: 3.4.0.639-dev
Build date: 17:47:13 Feb 21 2018 (1411)
Build from: https://github.com/dreamstalker/rehlds/commit/822df9a

ReGameDLL version: 5.7.0.295-dev
Build date: 15:10:22 Feb 19 2018
Build from: https://github.com/s1lentq/ReGameDLL_CS/commit/aa329db

      description      stat pend  file                    vers            src  load  unload
 [ 1] Reunion          RUN   -    reunion_mm.dll          v0.1.0.92       ini  Start Never
 [ 2] VoiceTranscoder  RUN   -    voicetranscoder.dll     v2017RC3        ini  ANY   ANY  
 [ 3] ReAuthCheck      RUN   -    reauthcheck_mm.dll      v0.1.6          ini  Start Never
 [ 4] WHBlocker        RUN   -    whblocker_mm.dll        v1.5.695        ini  Chlvl ANY  
 [ 5] Rechecker        RUN   -    rechecker_mm.dll        v2.5            ini  Chlvl ANY  
 [ 6] ReSemiclip       RUN   -    resemiclip_mm.dll       v2.3.9          ini  Chlvl ANY  
 [ 7] AMX Mod X        RUN   -    amxmodx_mm.dll          v1.8.3-Re       ini  Start ANY  
 [ 8] ReAPI            RUN   -    reapi_amxx.dll          v5.5.0.150-dev  pl7  ANY   Never
 [ 9] ReAimDetector    RUN   -    reaimdetector_amxx.dll  v0.2.1          pl7  ANY   Never
 [10] hackdetector     RUN   -    hackdetector_amxx.dll   v0.15.328.lite  pl7  ANY   ANY  
 [11] Ham Sandwich     RUN   -    hamsandwich_amxx.dll    v1.8.3-Re       pl7  ANY   ANY  
 [12] CSX              RUN   -    csx_amxx.dll            v1.8.3-Re       pl7  ANY   ANY  
 [13] CStrike          RUN   -    cstrike_amxx.dll        v1.8.3-Re       pl7  ANY   ANY  
 [14] Fun              RUN   -    fun_amxx.dll            v1.8.3-Re       pl7  ANY   ANY  
 [15] FakeMeta         RUN   -    fakemeta_amxx.dll       v1.8.3-Re       pl7  ANY   ANY  
15 plugins, 15 running

Currently loaded plugins:
       name                    version     author            file             status   
 [  1] Ultimate Bans           1.9         Souvik            UltimateBans.am  running  
 [  2] AMXX BanCfg             0.95        Safety1st         amx_bancfg.amxx  running  
 [  3] Auto BAN                1.0         kaboomkazoom & A  autoban.amxx     running  
 [  4] Admin Votes             1.8.3-Re    AMXX Dev Team     adminvote.amxx   running  
 [  5] UnBanmenu               1           AMX Team          unbanmenu.amxx   running  
 [  6] Ban Disconnected Menu   2.0.1       MRG               banmenu.amxx     running  
 [  7] Admin Base              1.8.3-Re    AMXX Dev Team     admin.amxx       running  
 [  8] Admin Commands          1.8.3-Re    AMXX Dev Team     admincmd.amxx    running  
 [  9] Admin Help              1.8.3-Re    AMXX Dev Team     adminhelp.amxx   running  
 [ 10] Slots Reservation       1.8.3-Re    AMXX Dev Team     adminslots.amxx  running  
 [ 11] Multi-Lingual System    1.8.3-Re    AMXX Dev Team     multilingual.am  running  
 [ 12] Menus Front-End         1.8.3-Re    AMXX Dev Team     menufront.amxx   running  
 [ 13] Commands Menu           1.8.3-Re    AMXX Dev Team     cmdmenu.amxx     running  
 [ 14] Players Menu            1.8.3-Re    AMXX Dev Team     plmenu.amxx      running  
 [ 15] Maps Menu               1.8.3-Re    AMXX Dev Team     mapsmenu.amxx    running  
 [ 16] Plugin Menu             1.8.3-Re    AMXX Dev Team     pluginmenu.amxx  running  
 [ 17] Anti Flood              1.8.3-Re    AMXX Dev Team     antiflood.amxx   running  
 [ 18] Admin Chat              1.8.3-Re    AMXX Dev Team     adminchat.amxx   running  
 [ 19] Scrolling Message       1.8.3-Re    AMXX Dev Team     scrollmsg.amxx   running  
 [ 20] Info. Messages          1.8.3-Re    AMXX Dev Team     imessage.amxx    running  
 [ 21] NextMap                 1.8.3-Re    AMXX Dev Team     nextmap.amxx     running  
 [ 22] Nextmap Chooser         1.8.3-Re    AMXX Dev Team     mapchooser.amxx  running  
 [ 23] TimeLeft                1.8.3-Re    AMXX Dev Team     timeleft.amxx    running  
 [ 24] Pause Plugins           1.8.3-Re    AMXX Dev Team     pausecfg.amxx    running  
 [ 25] Stats Configuration     1.8.3-Re    AMXX Dev Team     statscfg.amxx    running  
 [ 26] Restrict Weapons        1.8.3-Re    AMXX Dev Team     restmenu.amxx    running  
 [ 27] StatsX                  1.8.3-Re    AMXX Dev Team     statsx.amxx      running  
 [ 28] CS Misc. Stats          1.8.3-Re    AMXX Dev Team     miscstats.amxx   running  
 [ 29] CS Stats Logging        1.8.3-Re    AMXX Dev Team     stats_logging.a  running  
 [ 30] BecomeVIP Plus          2.0         OciXCrom          crx_becomevip.a  running  
 [ 31] VIP Extras #3           1.0         iPlay.bg (OciXCr  deageHEarmor.am  running  
 [ 32] WebSiteBot              1.0         Parker;           WebSiteBot.amxx  running  
 [ 33] Gag System              1.0         TheRedShoko @ AM  gag_system.amxx  running  
 [ 34] [ReAPI] No Team Flash   1.0         ReHLDS Team       noteamflash.amx  running  
 [ 35] High Ping Kicker (WON)  0.16.2      OLO/shadow        high_ping_kicke  running  
 [ 36] Chat Manager            3.6.4       OciXCrom          crx_chatmanager  running  
 [ 37] Reset Score             1.0         Silenttt          resetscore.amxx  running  
 [ 38] GHW Auto Message Displ  2.1         GHW_Chronic       GHW_Message_Dis  running  
 [ 39] [ReAPI] AdminFreeLook   1.0         ReHLDS Team       adminfreelook.a  running  
 [ 40] Bomb Countdown HUD Tim  0.2         SAMURAI           bcd_hudtimer.am  running  
 [ 41] Auto Demo Recorder      1.6         IzI               amx_demorecorde  running  
 [ 42] ReAimDetector API       0.2.1       ReHLDS Team       reaimdetector.a  running  
 [ 43] mute menu               1.0         cheap_suit        mutemenu.amxx    running  
 [ 44] MOTD Commands           1.1         OciXCrom          crx_motdcommand  running  
 [ 45] Anti Flood System Stab  4.0         kostov            Anti-Flood-Syst  running  
 [ 46] Descriptive 'Fire in t  0.1         VEN               descriptive_fir  running  
 [ 47] Ultimate SS             1.2         Hack              ultimate_ss.amx  running  
 [ 48] Headshot DeluXe         X.16.1.0    DanRaZor/dedi     headshot_deluxe  running  
 [ 49] Block Advertising       1.3         Alka              block_advertisi  running  
 [ 50] Show IP                 1.0         $uicid3           amx_ip.amxx      running  
 [ 51] No See Vars             0.1.0       ConnorMcLeod      no_see_vars.amx  running  
51 plugins, 51 running

I update everything with newest version. Normal I check  per two days for newest version.

**Comment by In-line** [comment]
ReGameDLL_CS and ReHLDS weren't updated for at least 12 days. It's likely doesn't related to them, if you updated every 2 days. So, if you updated 12 days ago, how did you notice crashes just now?

For first. Update outdated reamxmodx to amxmodx 1.8.3-dev. Nobody will support it.

**Comment by RealHumanWithHair** [comment]
Before I put last version of files there аре no have a problem. I checking and my server and there still no have crash from start the topic. 

Maybe someone somehow he crash it with whblock, but idk how.

I will update, but I thought ReAmxModX is current.

ty,
ps: if crash again i will share again :)

**Comment by csoyuncu** [comment]
yes there is crash problem at linux with version 695. When i removed bots, crash stopping but when i added bots, crash starts.
